packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.9.3
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* Reduces the update interval from 500ms to 100ms.
```

Comment:
I think this should be clearer about what update interval is changing

Reply:
thank you, I hope its more clear now.